lib/public/views/Runs/RunPerDataPass/RunsPerDataPassOverviewPage.js

```diff
@@ -47,11 +47,14 @@ export const RunsPerDataPassOverviewPage = ({ runs: { perDataPassOverviewModel }
 
     return h('', [
         h('.flex-row.justify-between.items-center', [
-            dataPass.name ? h('h2', `Good, physics runs of ${dataPass.name}`) : null,
+            dataPass.match({
+                Success: (payload) => h('h2', `Good, physics runs of ${payload.name}`),
+                Other: () => null,
+            }),
```

Comment:
As I already told you, `Other` should not be used to bypass the purpose of RemoteData! if there is an error you must handle it, if it's loading you should display it and so on!

Reply:
add title for each case